com.chartboost.mediation/Documentation/setup/ad-adapters.md

```diff
@@ -11,59 +11,120 @@ The Chartboost Mediation Unity SDK package contains samples with optional depend
 Each Chartboost Mediation Unity SDK sample contains a .xml file with dependencies for the adapter available platforms (Android/iOS). In order for such dependencies to be included into the final build the [Google's External Dependency Manager](https://github.com/googlesamples/unity-jar-resolver) is needed.
 The `EDM` makes sure to download and package any dependencies with your build.
 
-## ChartboostMediationDependencies.xml
+## Chartboost Mediation Adapters Window
 
-The `ChartboostMediationDependencies.xml` can be found in the `Package Manager/Sample` as seen in the following screenshot.
+Since Chartboost Mediation 4.X Adapters are no longer released at the cadence as the Chartboost Mediation SDK, this means is possible to receive adapter updates in between SDK updates. 
```

Comment:
```suggestion
Since Chartboost Mediation 4.X Adapters are no longer released at the cadence as the Chartboost Mediation SDK, it is now possible to receive adapter updates in between SDK releases. 
```